Homework_7/task_5.py

```diff
@@ -0,0 +1,17 @@
+"""
+    Task 5
+    Implement your own all function
+"""
+
+
+def custom_all(custom_list):
+    define_list = []
+    for el in range(len(custom_list)):
+        if custom_list[el]:
+            define_list.append(True)
+        else:
+            define_list.append(False)
```

Comment:
you do not need to append False. If any False appear you can just return it and that's all